libcxx/utils/CMakeLists.txt

```diff
@@ -7,9 +7,19 @@ add_custom_target(libcxx-generate-std-clang-module-header
   COMMENT "Generate the <__std_clang_module> header")
 
 add_custom_target(libcxx-generate-std-cppm-in-file
-  COMMAND "${Python3_EXECUTABLE}" "${CMAKE_CURRENT_SOURCE_DIR}/generate_std_cppm_in.py"
+  COMMAND
+        "${Python3_EXECUTABLE}"
+		"${LIBCXX_SOURCE_DIR}/utils/generate_libcxx_cppm_in.py"
+		"std"
   COMMENT "Generate the std.cppm.in file")
 
+add_custom_target(libcxx-generate-std-compat-cppm-in-file
+  COMMAND
+        "${Python3_EXECUTABLE}"
```

Comment:
Not a huge fan of the indentation here -- I think it would be more readable on a single line cause it would align better. Non-blocking.

Reply:
This seems to be done at other places in this file too, not consistently however. For now I keep it since it matches the most common case.